setup.py

```diff
@@ -358,6 +360,37 @@ def check_cudnn_version_and_warn(global_option: str, required_cudnn_version: int
             )
         )
 
+if PYTORCH_HOME is not None and os.path.exists(PYTORCH_HOME):
+    nvfuser_is_refactored = "nvfuser" in (
+        os.path.join(d) for d in os.listdir(os.path.join(PYTORCH_HOME, "third_party"))
+        if os.path.isdir(os.path.join(os.path.join(PYTORCH_HOME, "third_party"), d))
+    )
+    import nvfuser  # NOQA
+    print(PYTORCH_HOME)
+    include_dirs = [PYTORCH_HOME]
+    library_dirs = []
+    extra_link_args = []
+    if nvfuser_is_refactored:
+        include_dirs.append(os.path.join(PYTORCH_HOME, "third_party/nvfuser/csrc"))
+        library_dirs = nvfuser.__path__
+        extra_link_args.append("-lnvfuser")
```

Comment:
hmm, it looks like the file name is `nvfuser.so` not `libnvfuser.so`
```suggestion
        extra_link_args.append("-l :nvfuser.so")
```